README.md

```diff
@@ -161,15 +161,17 @@ const App = () => {
 ...
 return (
     <div className={config.THEME_CLASS}>
-      <KeycloakWrapper
-        clientId={config.KEYCLOAK_CLIENT_ID}
-        realm={config.KEYCLOAK_REALM}
-        url={config.KEYCLOAK_URL}
-        redirectUri={config.KEYCLOAK_REDIRECT_URI}
-        authenticationMethods={authenticationMethods}
-      >
-      ...
-      </KeycloakWrapper>
+      <LocalizationProvider customMessages={CUSTOM_MESSAGES}>
+        <OidcProvider
```

Comment:
I think we should also make the `post_logout_redirect_uri` as a configurable variable.

Reply:
Thanks! I've added the option to pass it as a config variable.